docs/alerts/README.md

```diff
@@ -0,0 +1,121 @@
+# Alerting on Synthetic Monitoring
```

Comment:
I think it would be worth adding at least a github link to this file in the README, because that gets published [here](https://grafana.com/grafana/plugins/grafana-synthetic-monitoring-app)

Reply:
updated, linked headline to plugin page

Reply:
Good idea! Sorry I meant to add a link to this doc from the root README in this repo, since that gets published to the plugins page. Although we might have to merge this before you can do that